app/views/posts/index.html.erb

```diff
@@ -1 +1,14 @@
-<h1>Here is a list of posts for a given user</h1>
\ No newline at end of file
+<body>
+    <%= render "partials/user_container", user: @users_find %>
+        <%= render "partials/post_container", post: @posts %>
+        <% comments = Comment.where(post_id: @posts.id) %>
+        <% comments.each do |comment| %>
+          <article class="comments-box">
+            <% user = User.find_by(id: comment.author_id).name %>
+            <p><strong><%=user%>: <%= comment.text %></strong></p>
+          </article>
+        <% end %>
```

Comment:
- [ ] It would be very nice if you use the `recent_comments` method to render 5 most recent comments to always show most recent comments and do your applications some work automatically :+1: